pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -170,10 +176,10 @@ func TestCRIListPodStats(t *testing.T) {
 		sandbox0, sandbox1, sandbox2, sandbox3, sandbox4, sandbox5,
 	})
 	fakeRuntimeService.SetFakeContainers([]*critest.FakeContainer{
-		container0, container1, container2, container3, container4, container5, container6, container7,
+		container0, container1, container2, container3, container4, container5, container6, container7, container8,
 	})
 	fakeRuntimeService.SetFakeContainerStats([]*runtimeapi.ContainerStats{
-		containerStats0, containerStats1, containerStats2, containerStats3, containerStats4, containerStats5, containerStats6, containerStats7,
+		containerStats0, containerStats1, containerStats2, containerStats3, containerStats4, containerStats5, containerStats6, containerStats7, containerStats8
```

Comment:
Missing comma: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/83504/pull-kubernetes-bazel-test/1180510732916101120

Running some of the tests locally is a good way to get feedback more quickly than Prow: https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#test